CRM/Upgrade/3.2.alpha1.msg_template/message_templates/event_offline_receipt_text.tpl

```diff
@@ -20,9 +19,7 @@ a link to a web page where you can complete your registration.{/ts}
 {ts}Your registration has been submitted.{/ts}
 
 {if $isPrimary}
-{ts}Once your registration has been reviewed, you will receive
-an email with a link to a web page where you can complete the
-registration process.{/ts}
+{ts}Once your registration has been reviewed, you will receive an email with a link to a web page where you can complete the registration process.{/ts}
 
```

Comment:
Well, options would be:

 a. Add the header to outbound emails.
 b. Implement automatic wrapping after the email has been composed but before it's sent.
 c. Revert the change (at least for plain-text).

I'm a little worried that changing the behavior (a or b) will open up more edge-cases (e.g. if a site has already customized an email template, and if we change the header, will it still look OK?). IMHO, it requires some thoughtful testing/migration. (I'd pencil in at least an hour if I were doing it -- maybe a day if things go sideways.) Doing this could be good -- we could wind up with better product/more confidence, but it's also a chunk of unplanned work.

If we can't put in the time, then I'd vote to revert the changes to the plain-text templates.

re: duplicate templates -- Agree that seems rather gratuitous.